examples/main-gen.c

```diff
@@ -116,6 +126,8 @@ int main(int argc, char *argv[])
 		jwt_add_grant(jwt, opt_claims[i].key, opt_claims[i].val);
 	}
 
+	ret = jwt_add_grants_json(jwt, opt_json);
```

Comment:
if (opt_json != NULL) {
        ret = jwt_add_grants_json(jwt, opt_json);
        /* Add check for error in ret */
    }